index.php

```diff
@@ -5,6 +5,9 @@
     <title>Upload photos for Yearbook 2016</title>
     <style media="screen">
       body {
+		  background-image:url(ybg.jpg);
+		  background-repeat:no-repeat;
+	       background-size:cover;
```

Comment:
@NeerajUnnikrishnan 

Fix the indentation of these three lines.

They should align with the `text-align` line's beginning.

***

cc @students-alumni-cell-iitkgp/web-team-members